src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/resource/CalculationController.kt

```diff
@@ -238,43 +192,7 @@ class CalculationController(
     calculationRequestId: Long,
   ): CalculationBreakdown {
     log.info("Request received return calculation breakdown for calculationRequestId {}", calculationRequestId)
-    return bookingService.getCalculationBreakdown(calculationRequestId)
-  }
-
-  @PostMapping(value = ["/{prisonerId}/validate"])
-  @PreAuthorize("hasAnyRole('SYSTEM_USER', 'RELEASE_DATES_CALCULATOR')")
-  @ResponseBody
-  @Operation(
-    summary = "Validates that the data for the given prisoner in NOMIS can be used to calculate a release date",
-    description = "This endpoint will validate that the data for the given prisoner in NOMIS can be supported by the " +
-      "calculate release dates engine",
-  )
-  @ApiResponses(
-    value = [
-      ApiResponse(responseCode = "200", description = "Validation job has run successfully, the response indicates if there are any errors"),
-      ApiResponse(responseCode = "401", description = "Unauthorised, requires a valid Oauth2 token"),
-      ApiResponse(responseCode = "403", description = "Forbidden, requires an appropriate role")
-    ]
-  )
-  fun validate(
-    @Parameter(required = true, example = "A1234AB", description = "The prisoners ID (aka nomsId)")
-    @PathVariable("prisonerId")
-    prisonerId: String,
-    @RequestBody
-    calculationUserInputs: CalculationUserInputs?
-  ): List<ValidationMessage> {
-    val sourceData = prisonService.getPrisonApiSourceData(prisonerId)
-    val validationMessages = validationService.validate(sourceData)
-    if (validationMessages.isEmpty()) {
-      try {
-        val booking = bookingService.getBooking(sourceData, calculationUserInputs ?: CalculationUserInputs())
-        calculationService.calculateReleaseDates(booking)
-      } catch (validationException: CrdCalculationValidationException) {
-        return listOf(ValidationMessage(validationException.validation))
-      }
-    }
-
-    return validationMessages
+    return calculationTransactionalService.getCalculationBreakdown(calculationRequestId)
```

Comment:
moved the main validation method into its own conrteoller with a separate end point